src/libstd/sync/mpsc/mod.rs

```diff
@@ -919,7 +919,7 @@ impl<T> Drop for Sender<T> {
 #[stable(feature = "mpsc_debug", since = "1.8.0")]
 impl<T> fmt::Debug for Sender<T> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Sender {{ .. }}")
+        f.pad("Sender {{ .. }}")
```

Comment:
The `{{` and `}}` need to be changed to `{` and `}` as this string is no longer being used as a format string.

Reply:
Good catch, thanks for the review!